app/src/main/res/menu/activity_main.xml

```diff
@@ -5,5 +5,23 @@
         android:id="@+id/action_search"
         android:icon="@drawable/search"
         android:title="@string/title_find_place"
-        app:showAsAction="always" />
+        app:showAsAction="always"/>
+
+    <item
+    android:id="@+id/location"
+    app:showAsAction="never"
+    android:title="@string/title_location"
+    />
+
+    <item
+        android:id="@+id/select_date"
+        app:showAsAction="never"
+        android:title="Select Date"
+        />
+
+    <item
+        android:id="@+id/setting"
+        app:showAsAction="never"
+        android:title="Setting"

```

Comment:
WARNING: [HardcodedText] Hardcoded string "Setting", should use `@string` resource
 Hardcoding text attributes directly in layout files is bad for several reasons:

* When creating configuration variations (for example for landscape or portrait)you have to repeat the actual text (and keep it up to date when making changes)

* The application cannot be translated to other languages by just adding new translations for existing string resources.

There are quickfixes to automatically extract this hardcoded string into a resource lookup.